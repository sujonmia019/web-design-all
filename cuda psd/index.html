<!DOCTYPE html>
<html lang="en">
<head>
	<!--meta data-->
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<!--web title-->
	<title>cuda - An awesome portfolio html website</title>
	<!--google fonts-->
	<link href="https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,700&display=swap" rel="stylesheet">
	<!--fontawesome link-->
	<script src="https://kit.fontawesome.com/68342a81a8.js" crossorigin="anonymous"></script>
	<!--vendors link-->
	<link rel="stylesheet" href="vendors/css/normalize.css">
	<link rel="stylesheet" href="vendors/css/grid.css">
	<link rel="stylesheet" href="vendors/css/animated.css">
	<!--resourse link-->
	<link rel="stylesheet" href="resoursec/css/style.css">
	<link rel="stylesheet" href="responsive.css">
	<!--favicon icon-->
	<link rel="shortcut icon" href="resoursec/img/logo.png">
</head>
<body>
	
<!--Start Header Area-->	
<header id="home">
	<div class="manu">
		<nav>
			<div class="row">
				<!--Main Logo-->
				<a href="#home"><img src="resoursec/img/logo.png" alt="cuda" class="min-logo"></a>
				<!--Start Navigation-->
				<ul class="min-nav">
					<li class="active"><a href="#home">home</a></li>
					<li><a href="#service">service</a></li>
					<li><a href="#meet">meet our</a></li>
					<li><a href="#skills">skills</a></li>
					<li><a href="#portfolio">portfolio</a></li>
					<li><a href="#about">about us</a></li>
					<li><a href="#contacts">contacts</a></li>
				</ul>
				<!--End Navigation-->
				<div class="mobile-menu">
					<span onclick="openNav()">&#9776</span>
					<div id="my-nav" class="overlary">
						<a href="javascript:void()" onclick="closeNav()" class="closebtn">&times;</a>
						<div class="overlary-content">
							<a onclick="closeNav()" href="#home">home</a>
							<a onclick="closeNav()" href="#service">service</a>
							<a onclick="closeNav()" href="#meet">meet our</a>
							<a onclick="closeNav()" href="#skills">skills</a>
							<a onclick="closeNav()" href="#portfolio">portfolio</a>
							<a onclick="closeNav()" href="#about">about us</a>
							<a onclick="closeNav()" href="#contacts">contacts</a>
						</div>
					</div>
				</div>
			</div>
		</nav>
	</div>
	
		<div class="row">
			<!--Start Button-->
			<div class="hero-text">
				<h1>Hi there! We are the new kids on the block and we build awesome websites and mobile apps.
				<a href="#contacts" class="btn btn-hero">work with us!</a>
			</div>
			<!--End Button-->
		</div>
</header>
<!--End Header Area-->	

<!--Start Services Area-->	
<section class="services-section js--services-section" id="service">
	<!--Start Services Hero Hadding-->
	<div class="row">
		<h2>SERVICES WE PROVIDE</h2>
		<p class="text-description">We are working with both individuals and businesses from all over the globe to create awesome websites and applications.</p>
    </div>
    <!--End Services Hero Hadding-->

    <!--Start Services Colum Section-->
	<div class="row">
		<div class="col span_1_of_4">
			<img src="resoursec/img/flug.png" alt="flug" class="service-icon">
			<h3>Branding</h3>
			<p>Lorem ipsum dolor sit amet, <br> consectetuer adipiscing elit,<br> sed diam nonummy nibh.</p>
		</div>
		<div class="col span_1_of_4">
			<img src="resoursec/img/pencil.png" alt="flug" class="service-icon">
			<h3>design</h3>
			<p>Sed ut perspiciatis unde<br> omnis iste natus error sit<br> voluptatem.</p>
		</div>
			<div class="col span_1_of_4">
			<img src="resoursec/img/setting.png" alt="flug" class="service-icon">
			<h3>development</h3>
			<p>At vero eos et accusamus et<br> iusto odio dignissimos qui<br> blanditiis praesentium.</p>
		</div>
		<div class="col span_1_of_4">
			<img src="resoursec/img/rocket.png" alt="flug" class="service-icon">
			<h3>rocket science</h3>
			<p>Et harum quidem rerum est<br> et expedita distinctio. Nam<br> libero tempore.</p>
		</div>		
	</div>
	<!--End Services Colum Section-->
</section>
<!--End Services Area-->

<!--Start Meet Team Area-->	
<section class="team-section" id="meet">
	<!--Start Team Hero Hadding-->
	<div class="row">
		<h2 class="shape">MEET OUR BEAUTIFUL TEAM</h2>
		<p class="text-description">We are a small team of designers and developers, who help brands with big ideas.</p>
	</div>
	<!--End Team Hero Hadding-->
	<div class="row">
		<!-- Start Team Colum-->
		<div class="col span_1_of_4">
			<img src="images/sujon1.jpg" alt="team images" class="team-img">
			<h3>ANNE HATHAWAY</h3>
			<a href="#" class="team-link">CEO / Marketing Guru</a>
			<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna.</p>
			<!--Team Social Icon-->
			<div class="social-icon">
				<ul>
					<li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
					<li><a href="#"><i class="far fa-envelope"></i></a></li>
					<li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
					<li><a href="#"><i class="fab fa-twitter"></i></a></li>
				</ul>
			</div>
		</div> 	
		<div class="col span_1_of_4">
			<img src="images/sujon1.jpg" alt="team images" class="team-img">
			<h3>ANNE HATHAWAY</h3>
			<a href="#" class="team-link">Creative Director</a>
			<p>Duis aute irure dolor in in voluptate velit esse cillum dolore fugiat nulla pariatur. Excepteur sint occaecat non diam proident.</p>
			<!--Team Social Icon-->
			<div class="social-icon">
				<ul>
					<li><a href="#"><i class="far fa-envelope"></i></i></a></li>
					<li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
					<li><a href="#"><i class="fab fa-twitter"></i></a></li>
				</ul>
			</div>
		</div> 
		<div class="col span_1_of_4">
			<img src="images/sujon1.jpg" alt="team images" class="team-img">
			<h3>ANNE HATHAWAY</h3>
			<a href="#" class="team-link">Lead Designer</a>
			<p>Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.</p>
			<!--Team Social Icon-->
			<div class="social-icon">
				<ul>
					<li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
					<li><a href="#"><i class="far fa-envelope"></i></a></li>
					<li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
					<li><a href="#"><i class="fab fa-twitter"></i></a></li>
				</ul>
			</div>
		</div> 
		<div class="col span_1_of_4">
			<img src="images/sujon1.jpg" alt="team images" class="team-img">
			<h3>ANNE HATHAWAY</h3>
			<a href="#" class="team-link">SEO / Developer</a>
			<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
			<!--Team Social Icon-->
			<div class="social-icon">
				<ul>
					<li><a href="#"><i class="far fa-envelope"></i></i></a></li>
					<li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
					<li><a href="#"><i class="fab fa-twitter"></i></a></li>
				</ul>
			</div>
		</div>
		<!-- End Team Colum--> 
	</div>
</section>
<!--End Meet Team Area-->	

<!--Start Skill Section-->	
<section class="skills-section" id="skills">
	<div class="row">
		<h2 class="shape">WE GOT SKILLS!</h2>
		<p class="text-description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
	</div>
	<div class="row">
		<div class="col span_1_of_4 box">
			<div class="chart" data-percent="85">85%</div>
			<h3>html / css</h3>
		</div>
		<div class="col span_1_of_4 box">
			<div class="chart" data-percent="85">85%</div>
			<h3>html / css</h3>
		</div>
		<div class="col span_1_of_4 box">
			<div class="chart" data-percent="85">85%</div>
			<h3>html / css</h3>
		</div>
		<div class="col span_1_of_4 box">
			<div class="chart" data-percent="85">85%</div>
			<h3>html / css</h3>
		</div>

	</div>
	
</section>
<!--End Skill Section-->

<!--Start Portfolio Section-->
<section class="portfolio-section" id="portfolio">
	<div class="row">
		<h2 class="p-shape">OUR PORTFOLIO</h2>
		<p class="text-description">Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet consectetur, adipisci velit, sed quia non numquam.</p>
	</div>
	<div class="row">
		<!--Start Portfolio Fillter Button-->
		<div class="p-fillter">
			<button type="button" data-filter="all">all</button>
			<button type="button" data-filter=".web">web</button>
			<button type="button" data-filter=".apps">apps</button>
			<button type="button" data-filter=".icons">icons</button>
		</div>
		<!--End Portfolio Fillter Button-->

		<!--Start Portfolio Fillter Colum-->
		<div class="row container">
			<div class="col span_1_of_2 box1 mix web">
				<img src="resoursec/img/mexitap1.png" alt="fillter images" class="fillter-img">
				<h4>Isometric Perspective Mock-Up</h4>
			</div>
			<div class="col span_1_of_2 box1 mix apps icons">
				<img src="resoursec/img/mexitap2.png" alt="fillter images" class="fillter-img">
				<h4>Time Zone App UI</h4>
			</div>
			<div class="col span_1_of_2 box1 mix icons">
				<img src="resoursec/img/mexitap3.png" alt="fillter images" class="fillter-img">
				<h4>Viro Media Players UI</h4>
			</div>
			<div class="col span_1_of_2 box1 mix apps web">
				<img src="resoursec/img/mexitap4.png" alt="fillter images" class="fillter-img">
				<h4>Blog / Magazine Flat UI Kit</h4>
			</div>
		</div>
		<!--End Portfolio Fillter Colum-->

		<!--Start Portfolio Button-->
		<div class="row">
			<a href="#" class="btn p-btn">LOAD MORE PROJECTS</a>
		</div>
		<!--End Portfolio Button-->
	</div>
</section>
<!--End Portfolio Section-->

<!--Start About Us Section-->	
<section class="about-us-section" id="about">
	<div class="row">
		<h2 class="about-text">WHAT POEPLE SAY ABOUT US</h2>
		<p class="text-description">Our clients love us!</p>
	</div>
	<!--Start About Us Colum Section-->
	<div class="row">
		<!--Testimonial-->
		<div class="col span_1_of_2">
			<div class="a-image">
				<img src="resoursec/img/about1.jpg" alt="about images" class="testimonial">
			</div>
			<div class="a-text">
				<p>“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”</p>
				<h4>Chanel Iman</h4>
				<span>CEO of Pinterest</span>
			</div>
		</div>
		<!--Testimonial-->
		<div class="col span_1_of_2">
			<div class="a-image">
				<img src="resoursec/img/about1.jpg" alt="about images" class="testimonial">
			</div>
			<div class="a-text">
				<p>“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta.”</p>
				<h4>ADRIANA LIMA</h4>
				<span>Founder of Instagram</span>
			</div>
		</div>
		<!--Testimonial-->
		<div class="col span_1_of_2">
			<div class="a-image">
				<img src="resoursec/img/about1.jpg" alt="about images" class="testimonial">
			</div>
			<div class="a-text">
				<p>“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta.”</p>
				<h4>ANNE HATHAWAY</h4>
				<span>Lead Designer at Behance</span>
			</div>
		</div>
		<!--Testimonial-->
		<div class="col span_1_of_2">
			<div class="a-image">
				<img src="resoursec/img/about1.jpg" alt="about images" class="testimonial">
			</div>
			<div class="a-text">
				<p>“Phasellus non purus vel arcu tempor commodo. Fusce semper, purus vel luctus molestie, risus sem cursus neque.”</p>
				<h4>EMMA STONE</h4>
				<span>Co-Founder of Shazam</span>
			</div>
		</div>
	</div>
	<!--End About Us Colum Section-->
</section>
<!--End About Us Section-->	

<!--Start Contact Us Section-->	
<section class="contact-section" id="contacts">
	<div class="row">
		<h2 class="contact-hero">GET IN TOUCH</h2>
		<p>1600 Pennsylvania Ave NW, Washington, DC 20500, United States of America. Tel: +8801872-339806</p>
	</div>
	<div class="row">
		<!--Contact Form-->
		<form action="https://formspree.io/webthemeupload@gmail.com" method="POST">
			<div class="row">
				<div class="col span_1_of_2">
					<input type="text" name="fname" placeholder="Your Name*" required>
				</div>
				<div class="col span_1_of_2">
					<input type="email" name="fname" placeholder="Your Email*" required>
				</div>
			</div>
			<div class="row">
				<textarea name="massage" cols="30" rows="10" placeholder="Your Massage*" require></textarea>
			</div>
			<div class="row">
				<input type="submit" value="send massages" class="btn c-send">
			</div>
		</form>
	</div>
</section>
<!--End Contact Us Section-->	

<footer class="footer-section">
	<div class="row">
		<ul>
			<li><a href="#">facebook</a></li>
			<li><a href="#">twitter</a></li>
			<li><a href="#">google+</a></li>
			<li><a href="#">linkedin</a></li>
			<li><a href="#">behance</a></li>
			<li><a href="#">bibbble</a></li>
			<li><a href="#">gitHub</a></li>
		</ul>
	</div>
</footer>
	








<!---js script--->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
<script src="https://code.jquery.com/jquery-3.4.1.js"></script>
<script src="vendors/js/jquery.easypiechart.min.js"></script>	
<script src="vendors/js/html5shiv.min.js"></script>	
<script src="vendors/js/respond.min.js"></script>	
<script src="vendors/js/selectivizr-min.js"></script>
<script src="vendors/js/jquery.sticky.js"></script>
<script src="vendors/js/animated.js"></script>
<script src="vendors/js/mixitup.min.js"></script>	
<script src="resoursec/js/min.js"></script>	

<!--Animated Custom js-->
<script>
    $(function() {
        $('.chart').easyPieChart({
            //your options goes here
            lineWidth:6,
        });
    });
</script>

</body>
</html>