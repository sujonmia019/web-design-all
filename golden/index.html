<!DOCTYPE html>
<html lang="en">
<head>
	<!--META FILES-->
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<!--TITLE-->
	<title>Global Media Centre</title>
	<!--GOOGLE FONTS-->
	<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
	<!--VENDORS FILES-->
	<link rel="stylesheet" href="vendors/normalize.css">
	<link rel="stylesheet" href="vendors/css/magnific-popup.css">
	<link rel="stylesheet" href="vendors/css/owl.carousel.min.css">
	<link rel="stylesheet" href="vendors/css/owl.theme.default.min.css">
	<link rel="stylesheet" href="vendors/css/meanmenu.min.css">
	<!--STYLESHEET-->
	<link rel="stylesheet" href="css/min.css" media="all">
	<link rel="stylesheet" href="responsive.css" media="all">
</head>
<body>
	<!--HEADER SECTION START-->
	<header class="fix">
		<div class="container">
			<div class="row">
				<div class="col-3 col-lg-12 col-md-12 col-sm-3">
					<!--MAIN LOGO-->
					<div class="logo">
						<a href="index.html"><img src="img/logo.png" alt="main-logo"></a>
					</div>
				</div>
				<div class="col-6 col-lg-9 col-md-9">
					<!--NAVIGATION START-->
					<div class="manu d-none">
						<nav id="mobile-menu">
							<ul>                                             
								<li><a href="#">Adventures</a>
									<ul class="sub-menu">
										<li><a href="#">sub manu 1</a></li>
										<li><a href="#">sub manu 2</a>
											<ul class="sub-menu">
												<li><a href="#">sub manu 1</a></li>
												<li><a href="#">sub manu 2</a></li>
												<li><a href="#">sub manu 3</a></li>
												<li><a href="#">sub manu 4</a>
													<ul class="sub-menu">
														<li><a href="#">sub manu 1</a></li>
														<li><a href="#">sub manu 2</a></li>
														<li><a href="#">sub manu 3</a></li>
														<li><a href="#">sub manu 4</a></li>
														<li><a href="#">sub manu 5</a></li>
													</ul>
												</li>
												<li><a href="#">sub manu 5</a></li>
											</ul>
										</li>
										<li><a href="#">sub manu 3</a></li>
										<li><a href="#">sub manu 4</a></li>
										<li><a href="#">sub manu 5</a></li>
									</ul>
								</li>
								<li><a href="#">Publishers</a></li>
								<li><a href="#">Agenciens</a></li>
								<li><a href="#">Company</a></li>
								<li><a href="#">Our Parthners</a></li>
								<li><a href="#">Contact</a></li>
							</ul>
						</nav>
					</div>
					<!--NAVIGATION END-->
				</div>
				<div class="col-3 col-lg-3 col-md-3 col-sm-9">
					<div class="header-right d-none">
						<div class="language d-none">
							<ul>
								<li><a href="#">EN <i class="fas fa-angle-down"></i></a></li>
							</ul>
						</div>						
						<div class="header-button">
							<a href="#">sign up</a>
						</div>					
					</div>
				</div>
				<div class="mobile-menu-active"></div>
			</div>
			<div class="container">
				<div class="row">
					<div class="hero-text">
						<h1>Global agency online marketing</h1>
						<a href="#">Sign Up right now</a>
					</div>
				</div>
			</div>
		</div>
	</header>
	<!--HEADER SECTION END-->

	<!--SERVICES SECTION START-->
	<section class="services-area fix">
		<div class="container">
			<div class="row">
				<div class="col-4 col-sm-12">
					<div class="single-services actives">
						<h2>Advertisers</h2>
						<p>That can be definitely said about the contemporary travel websites. </p>
						<a href="#" class="btn">join Now</a>
						<a href="#">Read More</a>
					</div>
				</div>
				<div class="col-4 col-sm-12">
					<div class="single-services">
						<h2>Publishers</h2>
						<p>That can be definitely said about the contemporary travel websites. </p>
						<a href="#" class="btn">join Now</a>
						<a href="#">Read More</a>
					</div>
				</div>
				<div class="col-4 col-sm-12">
					<div class="single-services">
						<h2>Agencies</h2>
						<p>That can be definitely said about the contemporary travel websites. </p>
						<a href="#" class="btn">join Now</a>
						<a href="#">Read More</a>
					</div>
				</div>
			</div>
		</div>
	</section>
	<!--SERVICES AREA END-->

	<!--WHY WE AREA START-->
	<section class="why-we-area fix">
		<div class="container">
			<div class="row">
				<div class="why-we-text">
					<h2>Why we?</h2>
					<p>That can be definitely said about the contemporary travel websites. </p>
				</div>
			</div>
			<div class="row">
				<div class="col-3 col-sm-12">
					<div class="choose-wapper">
						<img src="img/why-icons.png" alt="icon">
						<p>As the holidays approach, everything tends to get a little hectic. Stress levels increase, there seems to be more unfinished business to take care of than usual. And deadlines, which should always be met anyway,</p>
					</div>				
				</div>
				<div class="col-3 col-sm-12">
					<div class="choose-wapper">
						<img src="img/why-icons.png" alt="icon">
						<p>Even with jQuery’s popularity there’s no need to rely on a JS library for every project. Classic JavaScript works in all major browsers and it’s really all you need to build dynamic interfaces. </p>
					</div>
				</div>
				<div class="col-3 col-sm-12">
					<div class="choose-wapper">
						<img src="img/why-icons.png" alt="icon">
						<p>On the first day of Christmas my true love gave to me … a MacBook Pro with Creative Suite. Well, that’s not exactly how the holiday jingle goes, but if you are a web designer this list of gift suggestions is for you.</p>
					</div>
				</div>
				<div class="col-3 col-sm-12">
					<div class="choose-wapper">
						<img src="img/why-icons.png" alt="icon">
						<p>When custom label printer Consolidated Labels redesigned its website, their end goal was to increase customer conversions. Yet they liked their site, and didn’t see the need to make many changes. </p>
					</div>
				</div>
			</div>
		</div>
	</section>
	<!--WHY WE AREA END-->

	<!--WORK AREA START-->
	<section class="how-work-area fix">
		<div class="container">
			<div class="row">
				<div class="why-we-text">
					<h2>How we are working?</h2>
					<p>That can be definitely said about the contemporary travel websites.</p>
				</div>
			</div>
			<div class="row">
				<div class="col-2 col-md-4 col-sm-12">
					<div class="work actives">
						<h3>CPI</h3>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="work">
						<h3>CPa</h3>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="work">
						<h3>CPr</h3>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="work">
						<h3>CPc</h3>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="work">
						<h3>CPm</h3>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="work">
						<h3>rsm</h3>
					</div>
				</div>
			</div>
		</div>
	</section>
	<!--WORK AREA END-->

	<!--ADVERTISERS AREA START-->
	<section class="advertisers-area fix">
		<div class="container">
			<div class="row">
				<div class="advertiser-carousel owl-carousel">
					<div class="advertisers-item">
						<div class="col-6 col-sm-12">
							<div class="advertisers-test">
								<h2>advertisers</h2>
								<p>hat can be definitely said about the contemporary travel websites. Whether it is an page for a local hotel or a feature-heavy portal that belongs to big holiday company, it fantastic interfaces are user magnets.</p>
								<a href="#">read more</a>
								<div class="downloads">
									<img src="img/download.png" alt="images">
									<img src="img/download1.png" alt="images">
								</div>
							</div>
						</div>
						<div class="col-6 col-sm-12">
							<div class="mobile">
								<img src="img/mobile.png" alt="mobile images">
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>	
	</section>
	<!--ADVERTISERS AREA END-->

	<!--AGENCIES AREA START-->
	<section class="Agencies fix">
		<div class="container">
			<div class="row">
				<div class="col-12">
					<div class="Agencies-text">
						<h2>Agencies</h2>
						<p>Typography is an art combining perspective, technique and ingenuity. A great <br> design comes with great typography. </p>
					</div>
				</div>
			</div>
			<div class="row">
				<div class="col-6 col-sm-12">
					<div class="Agencies-wapper">
						<ul>
							<li>As the holidays approach, everything tends to get a little hectic. Stress levels increase, there seems to be more unfinished business to take care of than usual. And deadlines, which should always be met </li>
							<li>You can be the most creative and productive designer in the world, but it doesn’t mean anything without paid work. Designers can rely on repeat clients but it’s important to keep meeting new potential </li>
							<li>Today is your last day to shop the Cyber Monday sale and get 60% off! You can get the most popular tools for website creation at discounted prices. We have dropped the price of all our popular products </li>
						</ul>
					</div>
				</div>
				<div class="col-6 col-sm-12">
					<div class="Agencies-wapper">
						<ul>
							<li>As the holidays approach, everything tends to get a little hectic. Stress levels increase, there seems to be more unfinished business to take care of than usual. And deadlines, which should always be met </li>
							<li>You can be the most creative and productive designer in the world, but it doesn’t mean anything without paid work. Designers can rely on repeat clients but it’s important to keep meeting new potential </li>
							<li>Today is your last day to shop the Cyber Monday sale and get 60% off! You can get the most popular tools for website creation at discounted prices. We have dropped the price of all our popular products </li>
						</ul>
					</div>
				</div>	
			</div>
		</div>
	</section>
	<!--AGENCIES AREA END-->

	<!--VIDEO AREA START-->
	<section class="video-area fix" style="background-image: url(img/video-bg.jpg);background-size: cover; background-position: center center;">
		<div class="container">
			<div class="row">
				<div class="col-12">
					<div class="Publishers-hero">
						<h2>Publishers</h2>
						<p>hat can be definitely said about the contemporary travel websites. Whether it is an page for a local hotel or a feature-heavy portal that belongs to big holiday company, it fantastic interfaces are user magnets.</p>
					</div>
				</div>
				<div class="video-icons">
					<a class="video-popup" href="https://www.youtube.com/watch?v=uTKtJtNW0vM"><img src="img/video-icons.png" alt="video logo"></a>
				</div>
			</div>
		</div>
	</section>
	<!--VIDEO AREA END-->

	<!--ABOUT-US AREA START-->
	<section class="about-area fix py">
		<div class="container">
			<div class="row">
				<div class="col-12">	
					<div class="about-hero text-center">
						<h2>Fact about us</h2>
						<p>Whether you are creating for web or print, designing brochures or user interfaces, developing a brand identity or creating a logo, one of the key elements of a successful project is the typeface you are using. </p>
					</div>
				</div>
			</div>
			<div class="row">
				<div class="col-2 col-md-4 col-sm-12">
					<div class="about-wapper activate">
						<img src="img/about0.png" alt="about img">
						<p>Tips for Creating a Great Personal </p>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="about-wapper">
						<img src="img/about1.png" alt="about img">
						<p>How to Create Checkout Form Using HTML</p>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="about-wapper">
						<img src="img/about2.png" alt="about img">
						<p>WebAssembly Now in Browser Preview Mode</p>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="about-wapper">
						<img src="img/about3.png" alt="about img">
						<p>Adobe Introduces Project Felix, Sensei</p>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="about-wapper">
						<img src="img/about4.png" alt="about img">
						<p>A Design Workflow Comparison</p>
					</div>
				</div>
				<div class="col-2 col-md-4 col-sm-12">
					<div class="about-wapper">
						<img src="img/about5.png" alt="about img">
						<p>Top 5 Underutilized Web Performance </p>
					</div>
				</div>
			</div>	
		</div>
	</section>
	<!--ABOUT-US AREA END-->

	<!--GOOGLE-MAP AREA START-->
	<section class="map-area fix py" style="background-image: url(img/map.png);background-size: contain; background-position: center center;">
		<div class="container">
			<div class="row">
				<div class="map-address fix py">
					<span>phone:</span>
					<h3>+38 (098) 483 61 77</h3>

					<span>email:</span>
					<h3>info@globalmedia.com</h3>

					<span>skype:</span>
					<h3>globalmedia</h3>
				</div>
			</div>
		</div>
	</section>
	<!--GOOGLE-MAP AREA END-->

	<!--FOOTER AREA START-->
	<footer class="footer-area fix">
		<div class="container">
			<div class="row">
				<div class="copyright">
					<p>Copyright © 2019 Globalmedia. All Rights Reserved. Designmodo is a popular web design blog and shop.</p>
					<ul>
						<li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
						<li><a href="#"><i class="fab fa-twitter"></i></a></li>
						<li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
					</ul>
				</div>
			</div>
		</div>
	</footer>
	<!--FOOTER AREA END-->
	














	<!--JS FILES--> 
	<script src="https://code.jquery.com/jquery-3.4.1.js"></script>
	<script src="vendors/js/owl.carousel.min.js"></script>
	<script src="vendors/js/jquery.magnific-popup.min.js"></script>
	<script src="vendors/js/jquery.meanmenu.min.js"></script>
	<!--VENDORS JS-->
	<script src="vendors/js/html5shiv.min.js"></script>
	<script src="vendors/js/respond.min.js"></script>
	<script src="vendors/js/selectivizr-min.js"></script>
	<!--FONTAWESOME-->
	<script src="https://kit.fontawesome.com/68342a81a8.js"></script>
	<!--CUSTOM JS-->
	<script src="js/min.js"></script>
</body>
</html>